scripts/create_py_tags.py

```diff
@@ -8,296 +7,368 @@
 #
 # This script should be run in the top source directory.
 #
-# Parses all files given on command line for Python classes or functions and write
-# them into data/tags/std.py.tags (internal tagmanager format).
+# Parses all files in the directories given on command line for Python classes or functions and
+# write them into data/tags/std.py.tags (internal tagmanager format).
```

Comment:
looks like it's not in tagmanager format anymore, is it?

Reply:
Thank you for spotting, fixed.